platform/collaboration-tools/src/com/intellij/collaboration/auth/ui/login/TokenLoginInputPanelFactory.kt

```diff
@@ -80,12 +79,10 @@ class TokenLoginInputPanelFactory(
           .apply {
             onReset { component.text = "" }
           }
-          .component
 
         if (model is LoginTokenGenerator) {
           button(CollaborationToolsBundle.message("login.token.generate")) {
             model.generateToken(serverTextField.text)
-            IdeFocusManager.findInstanceByComponent(tokenField).requestFocus(tokenField, false)
```

Comment:
Not sure if dropping this is necessary

Reply:
I think that this is redundant because the focus should be on the Token password field in any case with or without the generate button but maybe I am missing something in here

Reply:
Yeah. I agree, for the main use-case focus will be returned to the field